hometown.html

```diff
@@ -0,0 +1,140 @@
+                    </p>
+                    <button class="btn"><a href="https://en.wikipedia.org/wiki/History_of_Chi%C8%99in%C4%83u"
+                            target="_blank">Read more</a></button>
+                </div>
+            </div>
+        </div>
+        <!-- Section Location -->
+        <div class="part2">
+            <div class="frame2">
+                <h2 id="loca">Location</h2>
+                <div class="loc">
+                    <div style="width: 100%"><iframe width="100%" height="300"
+                            src="https://maps.google.com/maps?width=100%&amp;height=600&amp;hl=en&amp;q=Chisinau%2C%20moldova+(My%20Hometown)&amp;ie=UTF8&amp;t=&amp;z=10&amp;iwloc=B&amp;output=embed"
+                            frameborder="0" scrolling="no" marginheight="0" marginwidth="0"><a
+                                href="https://www.maps.ie/map-my-route/">Draw map route</a></iframe></div><br />
+                </div>
+            </div>
+        </div>
+        <!-- Section Images -->
+        <div class="part2">
+            <div class="frame2">
+                <h2 id="imag">Images</h2>
+                <div class="images">
+                    <div class="pic_frame"><img class="pic" src="/assets/img/chisinau-by-night-kiri-photography-2.jpg"
+                            alt=""></div>
+                    <div class="pic_frame"><img class="pic" src="/assets/img/chisinau-by-night-kiri-photography-3.jpg"
+                            alt=""></div>
+                    <div class="pic_frame"><img class="pic" src="/assets/img/chisinau-by-night-kiri-photography-4.jpg"
+                            alt=""></div>
+                    <div class="pic_frame"><img class="pic" src="/assets/img/chisinau-by-night-kiri-photography-5.jpg"
+                            alt=""></div>
+                    <div class="pic_frame"><img class="pic" src="/assets/img/chisinau-by-night-kiri-photography-6.jpg"
+                            alt=""></div>
+                    <div class="pic_frame"><img class="pic" src="/assets/img/chisinau-by-night-kiri-photography-6.jpg"
+                            alt=""></div>
+                    <div class="pic_frame"><img class="pic" src="/assets/img/chisinau-by-night-kiri-photography-7.jpg"
+                            alt=""></div>
+                    <div class="pic_frame"><img class="pic" src="/assets/img/chisinau-by-night-kiri-photography-8.jpg"
+                            alt=""></div>
+                    <div class="pic_frame"><img class="pic" src="/assets/img/chisinau-by-night-kiri-photography-9.jpg"
+                            alt=""></div>
+                    <div class="pic_frame"><img class="pic" src="/assets/img/chisinau-by-night-kiri-photography-10.jpg"
+                            alt=""></div>
+
+                </div>
+            </div>
+        </div>
+        <!-- Section Insights -->
+        <div class="part2">
+            <div class="frame2">
```

Comment:
So sights here is inside frame2 inside part2.
Why don't we break it down in a section that contains two divs. One for the title and one for the content. Two rows for example. And then more divs inside for the sights. DIV DIV DIV DIV DIV DIV

Reply:
https://youtu.be/Vhh_GeBPOhs